server/preprocessing/other-scripts/test/params_openaire.json

```diff
@@ -0,0 +1,2 @@
+{"call_id":"FP7-PEOPLE-2013-IOF",
```

Comment:
As discussed, the relevant data for retrieving the project are funding stream & grant id and will be sent as parameters `funding_stream` and `grant_id`.